.licensure.yml

```diff
@@ -1,14 +1,57 @@
-author: ZomboDB, LLC
-ident: MIT
-email: zombodb@gmail.com
+change_in_place: true
 excludes:
   - Cargo.toml
-  - ".*\\.h"
-  - "\\.cargo/config"
+  - Cargo.lock
+  - LICENSE
+  - pgx-utils/assets/ansi.tmTheme
+  - ".envrc"
+  - ".gitignore"
+  - "flake\\..*"
+  - "logo.*"
+  - ".github/**/*"
+  - ".cargo/config"
   - "cargo-pgx/src/templates/*"
-  - ".github/workflows/*"
   - ".*\\.control"
   - ".*\\.md"
+  - ".*\\.nix"
   - ".*\\.yml"
-  - ".*\\.sh"
-  - load-order.txt
+licenses:
+  - files: any
+    ident: MIT
+    authors:
+      - name: ZomboDB, LLC and Technology Concepts & Design, Inc.
+        email: support@tcdi.com
+    template: |
+      Portions Copyright 2019-2021 ZomboDB, LLC. 
+      Portions Copyright 2021-[year] Technology Concepts & Design, Inc. <support@tcdi.com>.
+      All rights reserved. 
```

Comment:
I found previous artifacts of this statement in WebRTC's repo: https://webrtc.github.io/webrtc-org/license/software/

I did note that this may cause some confusion to some folks: https://github.com/PowerShell/PowerShell/issues/2129